.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+version: 2
+updates:
+
+  # Maintain dependencies for Java
+  - package-ecosystem: "maven"
+    directory: "/"
+    assignees: [b1ron]
+    open-pull-requests-limit: 5
+    schedule:
+      # Checks for updates 10 minutes before dance.
```

Comment:
The only reason that would make sense is if Dependabot's PRs will be approved and merged in 10 minutes – in the middle of the night.

Reply:
Yes understood, makes sense. I will change the time to early in the week.